File: 프로그래머스/lv1/12943. 콜라츠 추측/콜라츠 추측.js
function solution(num) {
    let answer = 0;
    while(answer <= 500) {
        if(num === 1) return answer
        if(num % 2 === 0) num /= 2
        else num = num * 3 + 1
        answer++
        console.log(num)
    }
    return -1
}

File: 프로그래머스/1/12943. 콜라츠 추측/콜라츠 추측.py
def solution(num):
    iter_num = 0
    while iter_num < 500:
        if num == 1:
            return iter_num
        if num % 2 == 0:
            num //= 2
        else:
            num = num * 3 + 1
        iter_num += 1
    return -1